cards/src/main/resources/cards/custom/group9/storyteller/basic/spell_magic_word_mishap.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "Magic Word Mishap",
+  "baseManaCost": 1,
+  "type": "SPELL",
+  "heroClass": "OLIVE",
+  "rarity": "FREE",
+  "description": "Deal $2 damage to a minion. If it survives, fully heal it.",
+  "targetSelection": "MINIONS",
+  "spell": {
+    "class": "ConditionalEffectSpell",
+    "condition": {
+      "class": "IsDeadCondition",
+      "invert": true
+    },
+    "spell1": {
+      "class": "DamageSpell",
+      "value": 2
+    },
+    "spell2": {
+      "class": "HealSpell",
+      "value": {
+        "class": "AttributeValueProvider",
+        "target": "TARGET",

```

Comment:
remove `"target": "TARGET"` it defaults to the target passed from conditional effect spell (which makes sense)